necta/src/bot/commands/find_by_tag.py

```diff
@@ -0,0 +1,24 @@
+from src import (
+    input_error,
+    NoNotesFound,
+    
+)
+from tabulate import tabulate
+from src.notes import Notes
+
+@input_error
+def find_by_tag(args, notebook: Notes):
+    tag, *_=args
+
+    notes = notebook.find_note_by_tag(tag)
+
+    if len(notes)==0:
```

Comment:
Тут треба трохи поправити пробіли з обох боків від оператора 
```suggestion
    if len(notes) == 0:
```